pkg/config/apply.go

```diff
@@ -146,7 +146,7 @@ func (a *ApplyImpl) SkipCleanup() bool {
 
 // SkipDeps returns the skip deps.
 func (a *ApplyImpl) SkipDeps() bool {
-	return a.ApplyOptions.SkipDeps
+	return a.GlobalOptions.SkipDeps || a.ApplyOptions.SkipDeps
```

Comment:
remove `SkipDeps` field in `ApplyOptions`. others are similar.

Reply:
for my understanding... this is because `ApplyImpl` embeds `GlobalImpl` which embeds `GlobalOptions`?